src/zframe.c

```diff
@@ -571,9 +571,9 @@ zframe_test (bool verbose)
 
     //  @selftest
     //  Create two PAIR sockets and connect over inproc
-    zsock_t *output = zsock_new_pair ("@inproc://zframe.test");
+    zsock_t *output = zsock_new_pair ("@tcp://127.0.0.1:9001");
```

Comment:
Ok, I got a segfault because I was running streq() on NULL. Duh!

Metadata doesn't get passed over inproc. Not sure why.